docs/content/usage/developer-workflow.md

```diff
@@ -231,14 +245,7 @@ to support multiplication, too.
     ...
 ```
 
-You can add the above lines to `calc.py`, or just download the
-modified function:
-
-```bash
-$ curl -Lo calc.py http://zzz
-```
-
-To deploy your changes, simply apply the specs again:
+You can add the above lines to `calc.py`. To deploy your changes, simply apply the specs again:
```

Comment:
cannot comment on non-pull-request code, please add syntax name at the first "```" of each code block or it will break the format.
```
 ```python
```